syfertext/__init__.py

```diff
@@ -97,3 +113,66 @@ def create(pipeline_name, tags: Set[str] = None, description: str = None):
 
 # Set the default owners of some classes
 # SubPipeline.owner = LOCAL_WORKER
+
+
+def save(pipeline_name: str, pipeline: "Pipeline", destination: Union["local"] = "local") -> None:
+    """Saves the pipeline and it's states to storage
+
+    Args:
+        pipeline_name (str): The name of the pipeline.
+        pipeline (Pipeline): The pipeline object itself
+        destination (Union): The location where to save that object, currently storage on local machine is implemented.
+
+    """
+
+    # Path to the home/SyferText directory
+    data_path = os.path.join(str(Path.home()), "SyferText")
```

Comment:
we can use appdirs https://pypi.org/project/appdirs/ to store in cache folder, instead of home directory